hw/top_englishbreakfast/data/top_englishbreakfast.hjson

```diff
@@ -140,23 +140,22 @@
     // likely external or generated from a voltage comparator
```

Comment:
QQ: how are the `domains` derived after this change? also, I think the comment above needs to be updated to reflect the updated data structure...

Reply:
this is all generated by the new `resets` class now. When the class is initialized, it loops through the modules and looks at what domain the module is sitting in and what resets it needs. The domain list is then created.

Hopefully this makes it less error prone in the future and we don't accidentally hardwire a block's reset to 0 again.